docs/data/base/components/menu/menu.md

```diff
@@ -17,61 +17,59 @@ import MenuUnstyled from '@mui/base/MenuUnstyled';
 import MenuItemUnstyled from '@mui/base/MenuItemUnstyled';
 ```
 
-The MenuUnstyled components can be used to create custom menus.
-It renders a list of items in a popup and allows mouse and keyboard navigation through them.
+You can use the `MenuUnstyled` component to create custom menus.
+It renders a list of items in a popup that users can navigate through with a mouse or keyboard.
 
-When not customized, the MenuItem renders a plain `ul` element.
+When not customized, the `MenuItemUnstyled` component renders a plain `<ul>` element.
```

Comment:
I think there's a mismatch here—wouldn't `MenuItemUnstyled` render an `<li>`? Should this say that `MenuUnstyled` renders a `<ul>'?

```suggestion
When not customized, the `MenuItemUnstyled` component renders a plain `<ul>` element.
```